src/stores/dashboardActions.ts

```diff
@@ -68,54 +70,59 @@ export const useBcrosDashboardActions = defineStore('bcros/dashboardActions', ()
     const isLegalType = businessStore.isLegalType
     const isBusiness = !!currentBusiness.value?.identifier
 
-    const { isStaffAccount } = useBcrosAccount()
     const { getFeatureFlag } = useBcrosLaunchdarkly()
     const legalType = currentBusiness.legalType
 
     switch (action) {
       case AllowableActionE.ADDRESS_CHANGE: {
         if (isEntityFirm()) {
-          return isAllowedToFile(FilingTypes.CHANGE_OF_REGISTRATION)
+          return isAllowedToFile(FilingTypes.CHANGE_OF_REGISTRATION) &&
+                 isAuthorized(AuthorizedActionsE.FIRM_CHANGE_FILING)
         }
-        return isAllowedToFile(FilingTypes.CHANGE_OF_ADDRESS)
+        return isAllowedToFile(FilingTypes.CHANGE_OF_ADDRESS) && isAuthorized(AuthorizedActionsE.ADDRESS_CHANGE_FILING)
```

Comment:
I agree with this -- yes, the action should not be visible if you're not authorized to do it.